dashboard/test/ui/browsers.json

```diff
@@ -2,68 +2,47 @@
 
 [
   {
-    "name": "Chrome72Win7",
-    "browserName": "chrome",
-    "version": "72",
-    "platform": "Windows 7",
-    "screenResolution": "1280x1024"
-  },
-  {
-    "name": "ChromeLatestWin7",
+    "name": "Chrome",
     "browserName": "chrome",
     "version": "latest",
     "platform": "Windows 7",
     "screenResolution": "1280x1024",
     "extendedDebugging": true
   },
   {
-    "name": "Chrome44Win7",
-    "browserName": "chrome",
-    "version": "44.0",
-    "platform": "Windows 7"
-  },
-  {
-    "name": "SafariYosemite",
+    "name": "Safari",
     "browserName": "Safari",
-    "platform": "OS X 10.10",
-    "version": "8.0"
+    "platform": "OS X 10.11",
+    "version": "10.0"
```

Comment:
Note: Updating to newer versions of Safari will require more thorough testing and refactoring of our UI tests and harness, because of several issues:
Safari 11.0 has a [bug where it does not handle click events on option elements](https://support.saucelabs.com/hc/en-us/articles/115003957233-Safari-11-with-Selenium-3-x-not-Handling-Click-Events-on-Option-Elements).
Safari 12 and up uses a new WebDriver that supports W3C endpoints rather than the JSON wire protocol. The W3C endpoint still has various compatibility issues with the current Selenium driver, so many tests fail. Sauce Labs has [beta support](https://wiki.saucelabs.com/display/DOCS/Selenium+W3C+Capabilities+Support+-+Beta) for W3C protocol but more testing/experimentation is needed to figure out if this will work with our current tests with modified configuration.